src/Android/Services/DeviceActionService.cs

```diff
@@ -752,6 +752,15 @@ public bool UsingDarkTheme()
             return false;
         }
 
+        public long GetActiveTime()
+        {
+            // Returns milliseconds since the system was booted, and includes deep sleep. This clock is guaranteed to
+            // be monotonic, and continues to tick even when the CPU is in power saving modes, so is the recommend
+            // basis for general purpose interval timing.
+            // ref: https://developer.android.com/reference/android/os/SystemClock#elapsedRealtime()
```

Comment:
This comment says returns ms, but looks like you are returning seconds?

Reply:
Correct, I standardized on seconds in the main app so platform-specific code converts accordingly.